src/granite/table.cr

```diff
@@ -44,7 +44,7 @@ module Granite::Table
     @@table_name = "{{table_name}}"
     @@primary_name = "{{primary_name}}"
     @@primary_auto = "{{primary_auto}}"
-    @@primary_type = "{{primary_type}}"
+    @@primary_type = {{primary_type}}
```

Comment:
I'm okay with this, and I agree it is a better paradigm. It is also a breaking change to the API.

Reply:
We could add a new method to get the actual type and leave the string one untouched?  If we wanted to avoid the breaking change.  However, I'm skeptical if it would be worth it?

Reply:
I have no problem incrementing the version, just want to me sure it's marked as a pull which is a breaking change.